Packs/SafeBreach/Integrations/SafeBreach/SafeBreach.yml

```
@@ -0,0 +1,2048 @@
+category: Deception & Breach Simulation
+description: "SafeBreach automatically executes thousands of breach methods from its extensive and growing Hacker’s Playbook™ to validate security control effectiveness. Simulations are automatically correlated with network, endpoint, and SIEM solutions providing data-driven SafeBreach Insights for holistic remediation to harden enterprise defenses."
+commonfields:
+  id: Safebreach
+  version: -1
+name: Safebreach
+display: Safebreach
+configuration:
+- display: Server URL
+  name: base_url
+  type: 0
+  required: true
+  additionalinfo: This is base URL for your instance.
+- display: API Key
+  name: api_key
+  type: 4
+  required: true
+  additionalinfo: This is API key for your instance, this can be created in safebreach user                       administration>APIkeys and then it must be saved as there is no way to view this again
```

for encrypted data it should be type 9 not 4, and please change its usage in the code file as well.
params.get("credentials", {}).get("identifier")
```suggestion
- display: API Key
  name: credentials
  type: 9
  section: Connect
  required: true
  additionalinfo: This is API key for your instance, this can be created in Safe Breach User                      Administration -> API keys, it must be saved as there is no way to view it again.
```

---

Using params.get("credentials", {}).get("identifier") will expose the API key to everyone. Therefore, I have opted to use params.get("credentials", {}).get("password") instead, ensuring that the API key remains concealed.